selfdrive/test/test_car_models.py

```diff
@@ -426,6 +426,16 @@ def get_route_log(route_name):
   HYUNDAI.KIA_STINGER,
   HYUNDAI.KONA,
   HYUNDAI.KONA_EV,
+  HYUNDAI.KONA_HEV,
+  HYUNDAI.GRANDEUR,
+  HYUNDAI.GRANDEUR_HEV,
+  HYUNDAI.KIA_NIRO_EV,
+  HYUNDAI.KIA_NIRO_HEV,
+  HYUNDAI.SONATA_H,
+  HYUNDAI.KIA_CADENZA,
+  HYUNDAI.KIA_CEED,
+  HYUNDAI.KIA_CADENZA_HEV,
+  HYUNDAI.KIA_SPORTAGE,
```

Comment:
Let's not add anything to the non-tested list. New models will require a test route before merging.